metadata-models/src/main/pegasus/com/linkedin/datajob/DataFlowInfo.pdl

```diff
@@ -38,4 +39,15 @@ record DataFlowInfo includes CustomProperties, ExternalReference {
     "queryByDefault": false
   }
   project: optional string
-}
\ No newline at end of file
+  
+  /**
+   * Optional list of jobs contained in this data flow
+   */
+  @Relationship = {
+    "/*": {
+      "name": "Contains",
+      "entityTypes": [ "dataJob" ]
+    }
+  }
+  jobs: optional array[DataJobUrn]
```

Comment:
Typically we no longer use strongly-typed URNs like this. 

You can just use `com.linkedin.common.urn` `Urn` type.